Public/html/portfolio.html

```diff
@@ -1,72 +1,103 @@
-            <div class="auth-links">
-                <a href="logout.html">Logout</a>
-            </div>
-        </div>
-    </nav>
+<body onload="loadPortfolioSummary(); loadPortfolio(); loadWatchlist(); loadRecentTransactions();">
+  <!-- Navigation Bar -->
+  <nav class="navbar">
+    <a href="#" class="logo">Investment App</a>
+    <div class="nav-links">
+      <a href="home.html">Home</a>
+      <a href="portfolio.html">My Portfolio</a>
+      <a href="notifications.html">Notifications</a>
+      <a href="trending.html">Trending</a>
+      <a href="#" onclick="logout()">Logout</a>
+    </div>
+  </nav>
 
-    <!-- Header Section -->
-    <header>
-        <h1>My Portfolio</h1>
-        <p>Track and manage your stock investments easily.</p>
-    </header>
+  <!-- Header Section -->
+  <header class="portfolio-header">
+    <h1>My Portfolio</h1>
+    <p>Manage your investments: Buy, sell, and track your performance.</p>
+  </header>
 
-    <!-- Main Content -->
-    <main>
-        <!-- Add Stock Button -->
-        <button class="add-stock-btn">Add Stock</button>
+  <!-- Portfolio Summary -->
+  <div id="portfolio-summary" class="summary-container">
+    <h2>Portfolio Summary</h2>
+    <p>Total Portfolio Value: $<span id="totalValue">0</span></p>
+    <p>Daily Profit/Loss: $<span id="dailyProfit">0</span></p>
+    <p>Overall Percentage Change: <span id="percentageChange">0</span>%</p>
+  </div>
 
-        <!-- Portfolio Table -->
-        <table class="portfolio-table">
-            <thead>
-                <tr>
-                    <th>Stock Symbol</th>
-                    <th>Shares</th>
-                    <th>Actions</th>
-                </tr>
-            </thead>
-            <tbody>
-                <tr>
-                    <td>AAPL</td>
-                    <td>50</td>
-                    <td>
-                        <button class="edit-btn">Edit</button>
-                        <button class="delete-btn">Delete</button>
-                    </td>
-                </tr>
-                <tr>
-                    <td>TSLA</td>
-                    <td>20</td>
-                    <td>
-                        <button class="edit-btn">Edit</button>
-                        <button class="delete-btn">Delete</button>
-                    </td>
-                </tr>
-            </tbody>
-        </table>
-    </main>
+  <!-- Watchlist -->
+  <div id="watchlist" class="watchlist-container">
+    <h2>Watchlist</h2>
+    <table>
+      <thead>
+        <tr>
+          <th>Stock Symbol</th>
+          <th>Current Price</th>
+          <th>Daily Change (%)</th>
+          <th>Actions</th>
+        </tr>
+      </thead>
+      <tbody id="watchlistTable"></tbody>
+    </table>
+  </div>
 
-    <!-- Footer -->
-    <footer>
-        © 2024 Investment App
-    </footer>
+  <!-- Portfolio Table -->
+  <div class="portfolio-container">
+    <button onclick="showAddStockForm()">Add Stock</button>

```

Comment:
Maybe centered or below the my portfolio section?

Reply:
Ok will do.